source/fundamentals/data-formats/codecs.txt

```diff
@@ -39,4 +39,11 @@ BsonTypeClassMap
 Custom Codec Example
 --------------------
 
-- Example, perhaps something like this https://stackoverflow.com/questions/45083885/decode-document-to-a-java-class-using-custom-mongo-codec
\ No newline at end of file
+- Methods to override
+- Encode: Describe parameters: BsonReader, EncoderContext
+- Decode: Describe parameters: BsonReader, DecoderContext
+- Example, perhaps something like this https://stackoverflow.com/questions/45083885/decode-document-to-a-java-class-using-custom-mongo-codec
```

Comment:
This looks reasonable.  You'd want to show how it's used, perhaps as a field in a POJO.